bin/post_compile

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash -x
```

Comment:
Why the shebang change? We should respect whatever system bash the user has setup. I also don't think we should use the `-x` here.